gov2/iam/README.md

```diff
@@ -2,266 +2,93 @@
-configured as described in
-[Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
-in the AWS SDK for Go Developer Guide.
-
-## Running the code
-
-### AccessKeyLastUsed/AccessKeyLastUsedv2.go
-
-This example retrieves when an IAM access key was last used, 
-including the AWS Region and with which service.
-
-`go run AccessKeyLastUsedv2.go -k KeyID`
-
-- _KeyID_ is the ID of the access key.
-
-The unit test accepts a similar value in _config.json_.
-
-### AttachUserPolicy/AttachUserPolicyv2.go
-
-This example attaches an Amazon DynamoDB full-access policy to an IAM role.
-
-`go run AttachUserPolicyv2.go -n ROLE-NAME`
-
-- _ROLE-NAME_ is the name of the role to which the policy is attached.
-
-The unit test accepts a similar value in _config.json_.
-
-### CreateAccessKey/CreateAccessKeyv2.go
-
-This example creates a new IAM access key for a user.
-
-`go run CreateAccessKeyv2.go -u USER-NAME`
-
-- _USER-NAME_ is the name of the user for whom the new key is created.
-
-### CreateAccountAlias/CreateAccountAliasv2.go
-
-This example creates an alias for your IAM account.
-
-`go run CreateAccountAliasv2.go -a ALIAS`
-
-- _ALIAS_ is the alias created for your account.
-
-The unit test accepts a similar value in _config.json_.
-
-### CreatePolicy/CreatePolicyv2.go
-
-This example creates an IAM policy.
-
-`go run CreatePolicyv2.go -n POLICY-NAME`
-
-- _POLICY-NAME_ is the name of the policy to create.
-
-The unit test accepts a similar value in _config.json_.
-
-### CreateUser/CreateUserv2.go
-
-This example creates an IAM user. 
-
-`go run CreateUserv2.go -u USER-NAME`
-
-- _USER-NAME_ is the name of the user to create.
-
-The unit test accepts a similar value in _config.json_.
-
-### DeleteAccessKey/DeleteAccessKeyv2.go
-
-This example deletes an IAM access key.
-
-`go run DeleteAccessKeyv2.go -k KeyID -u USER-NAME`
-
-- _KEYID_ is the access key to delete.
-- _USER-NAME_ is the name of the user deleting the key.
-
-The unit test accepts similar values in _config.json_.
-
-### DeleteAccountAlias/DeleteAccountAliasv2.go
-
-This example deletes an alias for your IAM account.
-
-`go run DeleteAccountAliasv2.go -a ALIAS`
-
-- _ALIAS_ is the account alias to delete.
-
-The unit test accepts a similar value in _config.json_.
-
-### DeleteServerCert/DeleteServerCertv2.go
-
-This example deletes an IAM server certificate.
-
-`go run DeleteServerCertv2.go -c CERTIFICATE-NAME`
-
-- _CERTIFICATE-NAME_ is the name of the server certificate to delete.
-
-The unit test accepts a similar value in _config.json_.
-
-### DeleteUser/DeleteUserv2.go
+## Code examples
 
-This example deletes an IAM user.
+### Scenario examples
+* [`common/`](common/) -- This scenario and examples demonstrate the creation of users, access keys, roles, policies, service-linked roles, and assuming a role in order to take on specific permissions from a policy.
```

Comment:
This scenario and its examples demonstrate how to create users, access keys, roles, policies, and service-linked roles. This scenario also demonstrates how to assume a role and take on specific permissions from a policy.